integration-tests/devtools/src/test/resources/__snapshots__/GrpcCodestartTest/testContent/src_main_proto_hello.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+option java_multiple_files = true;
+option java_package = "ilove.quark.us";
```

Comment:
while super fun, I think I would stick with "org.acme" .. OR, I would do something more standard: `dev.quarkus.grpc` or something. `ilove` is cute, but I don't like this projection of feelings.

Reply:
it's just for the test @ebullient, it's the "test" groupId

Reply:
https://github.com/quarkusio/quarkus/blob/main/independent-projects/tools/devtools-testing/src/main/java/io/quarkus/devtools/testing/codestarts/QuarkusCodestartTest.java#L44

Reply:
Fair enough. ;)